Count,Message,Traceback,Nodes
1,'record_dir',"  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 340, in run
    self.execute_next_task()
  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 373, in execute_next_task
    self.execute_task(self._task_queue.popleft())
  File ""/opt/anaconda3/envs/xn/lib/python3.11/site-packages/locust/user/task.py"", line 490, in execute_task
    task(self.user)
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 699, in mytask
    self.client.qianfan_request(
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 134, in qianfan_request
    request_meta = self._request_internal(context=context, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ""/Users/jianruitian/Documents/GitHub/bce-qianfan-sdk/python/qianfan/dataset/stress_test/qianfan_llm_load.py"", line 357, in _request_internal
    dir = GlobalData.data['record_dir']
          ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
",MacBook-Pro.local_9f2af7c43fe24be5adc44f28f02a70c0
